create_MyString_file.py

```diff
@@ -0,0 +1,109 @@
+class MyString:
+    def __init__(self,s):
+        self.string = s
+        self.length = MyString.len(self)
+
+    def __str__(self):
+        return self.string
+
+    def len(self):
+        count = 0
+        for char in self.string:
+            count += 1
+        return count
+
+    def add_string(self, other):
+        return '{}{}'.format(self.string,other)
```

Comment:
Expect return MyString type instead of string, just like when you add 2 fractions, you must return a fraction